spec/models/episode_spec.rb

```diff
@@ -1,18 +1,21 @@
 require 'rails_helper'
 
 RSpec.describe Episode, type: :model do
-  context 'with valid params' do
-    let(:valid_episode) { described_class.new(title: 'Name', date: Time.now) }
+  subject { described_class.new }
 
-    it { expect(valid_episode).to be_valid }
+  it 'is valid with valid attributes' do
```

Comment:
Check this for testing validations https://github.com/thoughtbot/shoulda-matchers